src/framework/vst/internal/vstmodulesrepository.cpp

```diff
@@ -128,6 +114,45 @@ void VstModulesRepository::addModule(const io::path& path)
     m_modules.emplace(io::basename(path).toStdString(), module);
 }
 
+audio::AudioResourceMetaList VstModulesRepository::modulesMetaList(const VstPluginType& type) const
+{
+    audio::AudioResourceMetaList result;
+
+    static auto hasNativeEditorSupport = []() {
```

Comment:
What do you think about transferring this lambda to VstConfiguration?

Reply:
Initially I was going to put it here, but I think we can avoid that restriction in the nearest future